src/main/java/net/syscon/elite/api/model/VisitWithVisitors.java

```diff
@@ -0,0 +1,32 @@
+package net.syscon.elite.api.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+
+import java.util.List;
+
+@ApiModel(description = "List of visitors for a visit")
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class VisitWithVisitors<V extends Visit> {
+    @ApiModelProperty(value = "List of visitors on visit", required = true)
+    @JsonProperty("visitors")
+    @NotBlank
```

Comment:
Is @NotBlank correct here? I thought it was only applicable to Strings?

Reply:
Very good spot, too much copy pasting. Fixed :)